docs/api-reference/next.config.js/redirects.md

```diff
@@ -13,6 +13,15 @@ description: Add redirects to your Next.js app.
   </ul>
 </details>
 
+<details>
+  <summary><b>Version History</b></summary>
+
+| Version   | Changes      |
+| --------- | ------------ |
+| `v10.1.0` | `has` added. |
```

Comment:
10.2 is when it was stable. Let's use that.

Reply:
https://nextjs.org/10-2

Reply:
```suggestion
| `v10.2.0` | `has` added. |
```